### PR TITLE
add Jan Sanchez to the AMA list

### PR DESCRIPTION
**Pull request by jansanchez**
Hi!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -59,7 +59,7 @@
 - [Acho Arnold](https://github.com/najela/ama) - Full Stack Web Developer with great passion in [Social Psychology](https://en.wikipedia.org/wiki/Social_psychology).
 - [Rob Furlong](https://github.com/robbbz/ama) - Maker of cool things with the web, freelancer.
 - [Honza Bittner](https://github.com/HoBi/ama) - High school student, web developer, Sass and Polymer ❤️'r.
-
```
Comment (arthurvr) [inline readme.md:62]: Don't remove this line ;)

Comment (jansanchez) [inline readme.md:62]: I'm sorry. done!

**Comment by jansanchez** [comment]
done!

**Comment by sindresorhus** [comment]
Welcome! :)

**Comment by sindresorhus** [comment]
Hey. Want to ask you a favor. Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=
https://github.com/sindresorhus/ama/issues?q=

**Pull request closed without merging**